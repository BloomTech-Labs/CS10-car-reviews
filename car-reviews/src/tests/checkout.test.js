import Checkout from '../components/UserPage/checkout';

it('renders Checkout component correctly', () => {
  const checkout = shallow(<Checkout />);
  expect(checkout).toMatchSnapshot();
});
