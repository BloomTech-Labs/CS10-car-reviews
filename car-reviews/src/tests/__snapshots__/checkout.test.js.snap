// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Checkout component correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Checkout />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "ComponentClass": "span",
      "amount": NaN,
      "className": "StripeCheckout",
      "currency": "USD",
      "description": undefined,
      "label": "Pay With Card",
      "locale": "auto",
      "name": undefined,
      "reconfigureOnUpdate": false,
      "stripeKey": "pk_test_gyGeHhPwsQ6mp7K1lSleMa4c",
      "token": [Function],
      "triggerEvent": "onClick",
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "ComponentClass": "span",
        "amount": NaN,
        "className": "StripeCheckout",
        "currency": "USD",
        "description": undefined,
        "label": "Pay With Card",
        "locale": "auto",
        "name": undefined,
        "reconfigureOnUpdate": false,
        "stripeKey": "pk_test_gyGeHhPwsQ6mp7K1lSleMa4c",
        "token": [Function],
        "triggerEvent": "onClick",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
